rewrite-test/src/main/kotlin/org/openrewrite/java/cleanup/SimplifyConstantIfBranchExecutionTest.kt

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@file:Suppress("StatementWithEmptyBody")
+@file:Suppress("StatementWithEmptyBody", "PointlessBooleanExpression")
```

Comment:
Because, yes, I'm fundamentally writing a bunch of "PointlessBooleanExpression" 😆